Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -1002,6 +1002,7 @@
 
 #if !defined(ENABLE_WK_WEB_EXTENSIONS) && (PLATFORM(MAC) || PLATFORM(MACCATALYST) || PLATFORM(IOS) || PLATFORM(IOS_SIMULATOR) || PLATFORM(VISION))
 #define ENABLE_WK_WEB_EXTENSIONS 1
+#define ENABLE_WK_WEB_EXTENSIONS_SIDEBAR 0
```

Comment:
This likely is best to be broken out on its own and be set to:

```
#if !defined(ENABLE_WK_WEB_EXTENSIONS_SIDEBAR)
#define ENABLE_WK_WEB_EXTENSIONS_SIDEBAR 0 && ENABLE_WK_WEB_EXTENSIONS
#endif
```

Reply:
Alrighty, this change should be in the PR now -- thanks!